src/poliastro/tests/test_frames.py

```diff
@@ -109,3 +112,20 @@ def test_icrs_body_position_to_planetary_frame_yields_zeros(body, frame):
         )
 
     assert_quantity_allclose(vector_result.xyz, [0, 0, 0] * u.km, atol=1e-7 * u.km)
+
+
+def test_round_trip_from_GeocentricSolarEcliptic_gives_same_results():
+    gcrs = SkyCoord("02h31m49.09s", "+89d15m50.8s", frame=GCRS(obstime=Time("J2000")))
```

Comment:
And also, I prefer that we avoid the use of `SkyCoord`, if possible.

Reply:
Done :)